README.md

```diff
@@ -12,53 +12,144 @@ A resource may classify its messages into groupings,
 
-### How to Contribute
-
-- **Issues** are for specific aspects of the resulting specification.
-  Discussions here should be focused, and stay on topic.
-  New issues must include a sufficient initial description to introduce themselves,
-  and may contain an exploration of possible solutions to them.
-- **Pull requests** help develop and define the eventual specification.
-  Accepting and merging a PR should not be taken as the final word on any topic,
-  but each change should be an improvement on the previous.
-  Each PR should only apply one change that may be squashed to the `main` branch.
-  Text content should use [semantic line breaks](https://sembr.org/).
-  A PR may close one or more issues, but this is not required.
-- **Discussions** are freeform, and may well refer to multiple issues or topics.
-  This is also the forum in which we should develop further our ways of working,
-  which may also include interactions and forums outside this repository,
-  such as occasional video calls.
-
-We welcome your participation and interest!
-
-## Syntax
-
-As currently specified, a message resource looks like this (syntax highlighting only approximate):
+This work was [presented](https://www.youtube.com/watch?v=ksgm_B-uUCU)
+at the [2024 Unicode Tech Workshop](https://www.unicode.org/events/utw/2024/).
+
+### Why?
+
+The work on MessageFormat 2 has identified the following challenges
+that go beyond or arise from the new message format,
+but which are not well addressed by existing resource formats:
+
+- The MessageFormat 2 syntax is naturally multi-line,
+  and multi-line values are not easy to use in many of the current resource formats.
+- Comments and metadata are important for translators,
+  but (mostly) not relevant for formatting.
+  Their attachment to messages needs to be well specified,
+  while being easy to read, write, and ignore.
+- Structured message metadata needs a common schema to be universally understood.
+- Many messages are composed of multiple localizable parts,
+  and it should be possible to express a compound message formed of multiple connected parts.
+- A purpose-built localization resource format should be well specified,
+  and designed from the ground up to work with multiple implementations.
+
+Some of these aspects are well supported by existing formats,
+but no one resource format serves all of the identified use cases.
+For instance:
+
+- Comments in Gettext .po files come in multiple types and [clearly assign] to the following entry,
+  but its representation of multi-line values is rather clumsy.
+- Fluent supports composability via [attributes] and [message references],
+  but its FTL format is tightly coupled with its own message format.
+- XLIFF is an [OASIS Standard],
+  but it's primarily a machine interchange format that is not designed for human authoring
+  and its message representation does not provide for message variants.
```

Comment:
Are plurals variants? If not, what's a variant in this context?

Reply:
Plurals are variants, yes, but messages can also vary e.g. based on gender.

Reply:
Just to confirm, there is no official support of plurals in the XLIFF standard? I'm finding contracdictory statements about support (and format) for XLIFF 2.

Reply:
Correct, XLIFF 2 does not ineherently support plurals. It does allow for a message to contain some completely separate syntax like ICU MessageFormat that does support plurals, and there are hacks that e.g. Apple's Xcode mapping does by generating separate localizable messages for each plural variant, but the resource format itself provides no such support.